src/utils/status/host/constants.js

```diff
@@ -1,5 +1,74 @@
 export const HOST_STATUS_READY = 'HOST_STATUS_READY';
+export const HOST_STATUS_DISCOVERED = 'HOST_STATUS_DISCOVERED';
+export const HOST_STATUS_OK = 'HOST_STATUS_OK';
+export const HOST_STATUS_PROVISIONED = 'HOST_STATUS_PROVISIONED';
+export const HOST_STATUS_DEPROVISIONED = 'HOST_STATUS_DEPROVISIONED';
+export const HOST_STATUS_REGISTERING = 'HOST_STATUS_REGISTERING';
+export const HOST_STATUS_INSPECTING = 'HOST_STATUS_VALIDATION_ERROR';
+export const HOST_STATUS_PREPARING_TO_PROVISION = 'HOST_STATUS_PREPARING_TO_PROVISION';
+export const HOST_STATUS_PROVISIONING = 'HOST_STATUS_PROVISIONING';
+export const HOST_STATUS_DEPROVISIONING = 'HOST_STATUS_DEPROVISIONING';
+export const HOST_STATUS_MAKING_HOST_AVAILABLE = 'HOST_STATUS_MAKING_HOST_AVAILABLE';
+export const HOST_STATUS_VALIDATION_ERROR = 'HOST_STATUS_VALIDATION_ERROR';
+export const HOST_STATUS_REGISTRATION_ERROR = 'HOST_STATUS_REGISTRATION_ERROR';
+export const HOST_STATUS_STARTING_MAINTENANCE = 'HOST_STATUS_STARTING_MAINTENANCE';
+export const HOST_STATUS_STOPPING_MAINTENANCE = 'HOST_STATUS_STOPPING_MAINTENANCE';
+export const HOST_STATUS_MAINTENANCE = 'HOST_STATUS_MAINTENANCE';
+
+export const HOST_STATUS_ALL_ERROR = [HOST_STATUS_REGISTRATION_ERROR, HOST_STATUS_VALIDATION_ERROR];
 
-// TODO(honza): fill out these values
-export const HOST_STATUS_ALL_ERROR = [];
 export const HOST_STATUS_ALL_WARN = [];
+
+export const HOST_STATUS_ALL_PROCESS = [
```

Comment:
I think this should be `HOST_STATUS_ALL_IN_PROGRESS`.

Reply:
Thanks!